api/src/digest/cron.ts

```diff
@@ -67,11 +70,14 @@ export class DigestCron {
     const projectsFromDataFolder = await this.dataService.listProjects();
 
     for (const project of projectsFromDataFolder) {
-      const [{ id: projectId }] = await this.projectsRepository.upsert({
+      const projectEntity = {
         ...project,
```

Comment:
this is a bug, lets only destruct the field we need (eg: repositories are leaked, it should not be exposed outside Cron context)

```suggestion
        ...project,
```